### DISCUSSION TITLE
starship didn't work on item2 of Mac

### DISCUSSION THREAD

**Discussion by leexiaominghub** [starship/starship#1460] [General]
intall:
```
curl -fsSL https://starship.rs/install.sh | bash
then:
which -a starship
/usr/local/bin/starship
echo $PATH
/Library/Java/JavaVirtualMachines/jdk1.8.0_221.jdk/Contents/Home/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin::/Users/lixiaoming/apache-maven-3.5.2/bin

cat ~/.zshrc
eval "$(starship init zsh)"
 cat ~/.bashrc
eval "$(starship init zsh)"
```

I try this :
```
source <("/usr/local/bin/starship" init zsh --print-full-init)
thread 'main' panicked at 'failed printing to stdout: Broken pipe (os error 32)', src/libstd/io/stdio.rs:822:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

what can I do?thanks

**Comment by matchai** [comment]
Hey @leexiaominghub 👋
This does seem very unusual. 🤔 

In case it's specifically the prompt printing that runs into problems, can you please try running the following, and creating a bug-report using the generated output:
```sh
starship bug-report
```

Otherwise, can you try setting the logging environment variable and running starship again:
```sh
STARSHIP_LOG=1 starship prompt
```

**Reply by leexiaominghub** [reply]
I have issue a bug as you tell me.Please have a look, thanks

**Comment by davidkna** [comment]
`.bashrc` is for `bash` and not `zsh`
Try `eval "$(starship init bash)"` instead of `eval "$(starship init zsh)" `in your `.bashrc` instead.

If your standard shell in iTerm is also bash (try `echo $SHELL`) you will also need to use `eval "$(starship init bash)"` instead there.

**Reply by leexiaominghub** [reply]
it works, thanks a lot
